uart/uart_posix.c

```diff
@@ -56,14 +67,63 @@ typedef struct {
 // Set time-out on 30 miliseconds
 const struct timeval timeout = {
   .tv_sec  =     0, // 0 second
-  .tv_usec = 30000  // 30000 micro seconds
+  .tv_usec = 300000  // 300000 micro seconds
```

Comment:
Is this  change related to the TCP  port? Why is it required? How does it affect the USB connection?

Reply:
It's needed because TCP may take longer that direct USB connection. As long as USB connection works smoothly it will have no effect. The only effect is that it will wait longer before declaring failure in case of flaky communication